### DISCUSSION TITLE
Print head freezes and then continues

### DISCUSSION THREAD

**Discussion by jake161** [supermerill/SuperSlicer#1668] [Q&A]
During a specific print, the print head will freeze and then start up again. It starts to ooze while it's frozen which leaves some surface imperfection. Does anyone know why this is? I have retraction disabled so that could explain the ooze, but I'm wondering why it's freezing. I'll include the gcode file sliced on SuperSlicer 2.3.56.9
[LONGBOI_Coiled_v2_with_hat_0.2mm_PLA_AKLP_2h4m.txt](https://github.com/supermerill/SuperSlicer/files/7392217/LONGBOI_Coiled_v2_with_hat_0.2mm_PLA_AKLP_2h4m.txt)

**Comment by supermerill** [comment]
see https://github.com/supermerill/SuperSlicer/issues/1632
